iot-hub/Quickstarts/InvokeDeviceMethod/Parameters.cs

```diff
@@ -0,0 +1,24 @@
+using CommandLine;
+
+namespace InvokeDeviceMethod
+{
+    /// <summary>
+    /// Command line parameters for the InvokeDeviceMethod sample
+    /// </summary>
+    internal class Parameters
+    {
+        [Option(
```

Comment:
For consistency with other sample's parameters files, please match the option alias and name: https://github.com/Azure-Samples/azure-iot-samples-csharp/blob/main/iot-hub/Samples/service/DigitalTwinClientSamples/Thermostat/Parameters.cs

Reply:
I updated the parameter names, but the shorthand flags in the other sample (-c and -d) don't work for me. I believe because they're reserved terms for dotnet already? I went with -s for string and -i for Id, but I'm happy to take suggestions for better options.

Reply:
You can pass [parameters to the application](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-run) and not worry about conficts with `dotnet run -- -p <cs> -d <deviceId>

Reply:
Got it. I wasn't using the `--` signifier to prefix my parameters. Now it works with -c and -d. Pushing that commit momentarily.